docs/usage-guide/topics/ch06-security-policies.md

```diff
@@ -118,3 +118,17 @@ Note that legacy SHA-1 algorithms are not supported in TLS1.3. Legacy SHA-1 algo
 |   20190802    |     X     |     X     |        |
 |   20200207    |     X     |     X     |    X   |
 |    rfc9151    |           |     X     |        |
+
+### Certificate Preferences
+
+The security policy struct contains three fields related to certificate preferences. These function differently from the other security policy fields because they are not sent to the peer as part of negotiation. Instead they are a set of rules that the endpoint will enforce locally.
+
+`certificate_signature_preferences` determine which certificate signatures are allowed in certificates. This field is an allowlist of `(signature_algorithm, digest)` tuples. Allowed `signature_algorithm`s are ECDSA, RSA PKCS v1.5, and RSA PSS. Note that s2n-tls does not currently provide support for restricting the digest types of RSA PSS signature, so a security policy using certificate signature preferences must either allow all digests with RSA PSS signatures or disallow RSA PSS signatures entirely.
```

Comment:
would it make sense to have these be the actual values, as in `S2N_SIGNATURE_ECDSA`, `S2N_SIGNATURE_RSA_PSS_PSS`, `S2N_SIGNATURE_RSA_PSS_RSAE`